packages/babel-plugin-transform-es2015-modules-commonjs/test/fixtures/misc/module-exports/actual.js

```diff
@@ -0,0 +1,22 @@
+import "foo";
```

Comment:
Do we have a test with a locally defined `exports`?

Reply:
I don't think we do. I'll add some.

Reply:
Done